### PR TITLE
HW_1 Pull Request

### PR DESCRIPTION
**Pull request by falese**
Turning in my homework!

Thanks!

### CODE REVIEW

**Comment by tgaff** [comment]
Wow you got pretty creative with this.  Looks great!  I'm going to add a few comments as pointers for next time, but nothing serious.

File: SeanFalese-falese/hw_2/reverse_it.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+drink = gets.chomp
+
+goodbye = "Tight, #{name.capitalize}, that'll be $#{drink.length} dollars"
+
+
+
+
+puts greeting
+if
```
Comment (tgaff) [inline SeanFalese-falese/hw_2/reverse_it.rb]: try to get the condition for your `if` on the same line to make it easier to read.

File: SeanFalese-falese/hw_2/reverse_it.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+end
+
+
+
+puts goodbye
+
+
+
+else
```
Comment (tgaff) [inline SeanFalese-falese/hw_2/reverse_it.rb]: This else is way far down and at first glance is a little confusing.  Part of that is white-space and part of it is indentation.  You can help clarify to others what `end` or `else` goes with what `def` or `if` by using two-space indentation. Ex:
```ruby
if name == 'Sean'
  if today == 'Friday'
    puts 'hey Sean, have a beer - on the house'
  else
    puts 'hey Sean, welcome back'
  end
else
  puts "I don't know you"  
end

Comment (falese) [inline SeanFalese-falese/hw_2/reverse_it.rb]: Thanks for the feedback!  I'll cleanup before class!

**Pull request merged**